ncrystal_python/src/NCrystal/ncmat2endf.py

```diff
@@ -321,7 +352,7 @@ def __init__(self, ncmat_fn, temperatures, elastic_mode,
         verbosity : integer
             Level of verbosity for the output
         """
-        self._temperatures = np.sort(np.asarray(temperatures))
+        self._temperatures = _np.sort(_np.asarray(temperatures))
```

Comment:
I would add `dtype=float` in the `asarray(..)` call, since it will force the correct type even if the caller would supply the temperatures as integers (which might happen easily in the Python API).

Reply:
Agree. I am also moving this out to the top level function `ncmat2endf`.